integration-tests/search.test.ts

```diff
@@ -515,9 +515,12 @@ describe('search', () => {
     });
 });
 
-describe('searches with patient permission levels', () => {
+describe('test searches with different scopes', () => {
     let adminClient: AxiosInstance;
```

Comment:
This can be moved to the `beforeAll` context and made `const`

Reply:
It's also used in `afterAll()`, which is why I kept it out